src/Publication/Publication.service.js

```diff
@@ -1,23 +1,27 @@
 import { Publication } from './Publication.model'
-import { Parcel } from '../Parcel'
-import { Estate } from '../Estate'
-import { ASSET_TYPES } from '../shared/asset'
+import { ASSETS } from '../Asset'
+import { PUBLICATION_ASSET_TYPES } from '../shared/publication'
 
 export class PublicationService {
   constructor() {
     this.Publication = Publication
-    this.Parcel = Parcel
-    this.Estate = Estate
   }
 
-  getModelFromAssetType(assetType) {
+  getPublicableAssetFromType(assetType) {
     if (!this.Publication.isValidAssetType(assetType)) {
       throw new Error(`Invalid publication asset_type "${assetType}"`)
     }
 
-    return {
-      [ASSET_TYPES.parcel]: this.Parcel,
-      [ASSET_TYPES.estate]: this.Estate
-    }[assetType]
+    return ASSETS[assetType]
+  }
+
+  getPublicableAssets(assetType) {
+    const publicableAssets = {}
+
+    for (const key in PUBLICATION_ASSET_TYPES) {
+      publicableAssets[key] = ASSETS[key]
+    }
+
+    return publicableAssets
```

Comment:
This return value is an `object` but it's expected to be an `array` [here](https://github.com/decentraland/marketplace/pull/492/files#diff-8b7ae6d8fe218adc457ca3f6733286c1R13) and [there](https://github.com/decentraland/marketplace/pull/492/files#diff-8b7ae6d8fe218adc457ca3f6733286c1R17)

Reply:
good catch! didn't test my last changes correctly